pkgs/applications/science/math/giac/default.nix

```diff
@@ -91,7 +91,8 @@ stdenv.mkDerivation rec {
     license = licenses.gpl3Plus;
     ## xcas is buildable on darwin but there are specific instructions I could
     ## not test
-    platforms = platforms.linux;
+    ## ^^ not sure what this means
```

Comment:
@symphorien do you remember what you meant by that?

Reply:
Yeah that was a note that I wanted to discuss that comment above about `xcas`

Reply:
OSX is supported as per https://www-fourier.ujf-grenoble.fr/~parisse/giac_compile.html
but the compilation process is said to be different: quote:
> Notes for compilation on Mac OS X. For recent versions of giac (>=0.9.x) on Mac OS X.6 run the script mkosx.

So I thought the derivation would not work as such on osx and would require more work.

Reply:
I xcas works as is on darwin, you can just remove the comment.